projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-wpcom-offline-subscription-service.php

```diff
@@ -46,7 +46,13 @@ public function subscriber_can_receive_post_by_mail( $user_id, $post_id ) {
 		$previous_user = wp_get_current_user();
 		wp_set_current_user( $user_id );
 
-		$access_level       = get_post_meta( $post_id, '_jetpack_newsletter_access', true );
+		$access_level = get_post_meta( $post_id, '_jetpack_newsletter_access', true );
+
+		if ( null === $access_level || self::POST_ACCESS_LEVEL_EVERYBODY === $access_level ) {
```

Comment:
Just do `! $access_level` instead of `empty( $access_level )` if that's the direction you want to go with it.

`empty( $access_level )` does the same thing as `( ! isset( $access_level ) || ! $access_level )`, but we know `$access_level` is already set because we just set it above.